modules/rosa-hcp-vpc-terraform.adoc

```diff
@@ -58,10 +58,8 @@ $ terraform apply rosa.tfplan
 ----
 $ export SUBNET_IDS=$(terraform output -raw cluster-subnets-string)
 ----
-
-.Verification
-
-* You can verify that the variables were correctly set with the following command:
++
+** Verify that the variables were correctly set with the following command:
```

Comment:
```suggestion
* Verify that the variables were correctly set by running/entering the following command:
```
Nit for internationalization. +1 to Janelle's comments. I think you only need one `*` here, right?

Reply:
@michaelryanpeter can you check the PR now? We restructured all of the optional ones. I thought it would be a `**` since it's supposed to be a sub-bullet to 2. - it's a one step procedure if you did the optional numbered step.